### PR TITLE
Fix styleguide forms

### LINKED ISSUE #1884: Fix broken styleguide forms

### PR DESCRIPTION
**Pull request by mparke**
closes #1884 

and fixes various css regressions due to various recent changes in manager and docs

Before:

screen shot 2017-06-14 at 4 09 41 pm

After:

screen shot 2017-06-14 at 4 01 51 pm
screen shot 2017-06-14 at 4 01 56 pm

Follow ups:

- https://github.com/linode/manager/issues/2077
- https://github.com/linode/manager/issues/2078
- https://github.com/linode/manager/issues/2079
- https://github.com/linode/manager/issues/2080

### CODE REVIEW

File: components/forms/FormSummary.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  return (
+    <div className="FormSummary">{content}</div>
+  );
+}
+
+FormSummary.propTypes = {
+  errors: PropTypes.object.isRequired,
+  success: PropTypes.string,
+};
```
Comment (eatonphil) [inline components/forms/FormSummary.js]: I kept this out of here because this stuff is only needed in the Manager right now.

Comment (eatonphil) [inline components/forms/FormSummary.js]: But I guess if it's for the styleguide...

Comment (mparke) [inline components/forms/FormSummary.js]: yeah, page broke without being able to show errors so I moved it

**Comment by abemassry** [comment]
image
Links back to /linodes

**Comment by abemassry** [comment]
@abemassry not a problem

**Pull request merged**